packages/shared/environment/src/EnvironmentError.ts

```diff
@@ -3,18 +3,13 @@ import {
   ErrorType
 } from "@gravitywelluk/error";
 
-export enum EnvironmentErrorCode {
-  MissingVariables = "missing_variables",
-  MissingAWSSecrets = "missing_aws_secrets"
-}
-
 /**
  * Environment Error class
  *
  * Custom APIError for an Environment error.
  */
-export class EnvironmentError extends APIError<EnvironmentErrorCode> {
-  constructor(msg: string, type?: ErrorType, code?: EnvironmentErrorCode, param?: Record<string, unknown>) {
-    super(msg, type, code, param);
+export class EnvironmentError extends APIError {
```

Comment:
Not sure this class is needed at all anymore as we aren't adding any codes or context